docs/UserGuide.md

```diff
@@ -882,28 +936,29 @@ If all else fails, you can try [uploading the image via GUI](#adding-by-graphica
+|-------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Open customer/Switch to Customer Tab**](#viewing-a-customer-opencus)                         | `opencus INDEX`<br> e.g., <code>opencus <em>2</em></code>, <code>opencus</code>                                                                                                                                                                                                                                                       |                                                                                              |
+| [**Add customer**](#adding-a-customer-addcus)                                                   | `addcus n/NAME p/PHONE_NUMBER e/EMAIL [a/ADDRESS] [t/TAG]...`  <br> e.g., <code>addcus n/<em>John Doe</em> p/<em>98765432</em> e/<em>johnd@example.com</em> a/<em>John street, block 123, #01-01</em> t/<em>animal cartoons</em></code>                                                                                               |
+| [**Edit customer**](#editing-a-customer-editcus)                                                | `editcus INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...` <br> e.g., <code>editcus <em>1</em> p/<em>91234567</em> e/<em>johndoe@example.com</em></code>                                                                                                                                                                     |
+| [**Delete customer**](#deleting-a-customer-delcus)                                              | `delcus INDEX`<br> e.g., <code>delcus <em>14</em></code>                                                                                                                                                                                                                                                                              |
+| [**Open Commission/Switch to Commission Tab**](#viewing-a-commission-opencom)                   | `opencom INDEX`<br> e.g., <code>opencom <em>14</em></code>, <code>opencus</code>                                                                                                                                                                                                                                                      |
```

Comment:
here too, `opencom [INDEX]`?

Reply:
Oh wait, I forgot this during merge conflict, thanks for noticing.